app/controllers/users_controller.rb

```diff
@@ -1,15 +1,24 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, except: [:show, :new, :create]
+  before_action :correct_user, only: [:edit, :update]
+  before_action :admin_user, only: :destroy
+  before_action :load_user, only: [:show, :edit, :update, :destroy]
   def new
     @user = User.new
   end
 
+  def index
+    @users = User.paginate page: params[:page],
+      per_page: Settings.users.index.per_page
```

Comment:
Mục đích phân trang là gì

Reply:
Dạ vì khi có lượng user lớn thì việc hiển thị trên cùng 1 trang 1 lúc sẽ khiến cho trang rất dài và khó đọc, và việc load 1 lúc toàn bộ user trên trang sẽ tốn thời gian và tài nguyên. 
Phân trang sẽ trả về 1 list user với số user mỗi trang ta định sẵn, vd trang thứ 3 với 10 user mỗi trang sẽ là user thứ 21-30